packages/sds/src/sds.spec.ts

```diff
@@ -471,4 +471,80 @@ describe("MessageChannel", function () {
       expect(possiblyAcknowledged.length).to.equal(messagesA.length);
     });
   });
+
+  describe("Sync messages", () => {
+    beforeEach(() => {
+      channelA = new MessageChannel(channelId);
+      channelB = new MessageChannel(channelId);
+    });
+
+    it("should be sent with empty content", () => {
+      channelA.sendSyncMessage((message) => {
+        expect(message.content?.length).to.equal(0);
+        return true;
+      });
+    });
+
+    it("should not be added to outgoing buffer, bloom filter, or local log", () => {
+      channelA.sendSyncMessage();
+
+      const outgoingBuffer = (channelA as any).outgoingBuffer as Message[];
+      expect(outgoingBuffer.length).to.equal(0);
+
+      const bloomFilter = getBloomFilter(channelA);
+      expect(
+        bloomFilter.lookup(MessageChannel.getMessageId(new Uint8Array()))
```

Comment:
the functionality is good, however, here we check if an empty content message ID exists in the filter - perhaps we can fetch an existing message ID like:
```ts
expect(bloomFilter.lookup(someRealMessageId)).to.equal(true/false)
```